common/locales/de.json

```
@@ -83,12 +83,18 @@
       "fields": {
         "category": "Block Kategorie",
         "location": "Ort",
+        "progressLabel": "Fortschritt",
```

```suggestion
        "progressLabel": "Status",
```
https://github.com/ecamp/ecamp3/pull/3405#issuecomment-1501158472